udf/num_weeks_gte_n_from_bits.sql

```diff
@@ -0,0 +1,13 @@
+/*
+
+Returns the number of weeks with >=n nonzero bits, given an integer representing
+a bit array for the last 4 weeks.
+
+*/
+CREATE OR REPLACE FUNCTION udf.num_weeks_gte_n_from_bits(b BYTES, n INT64) AS (
```

Comment:
```suggestion
CREATE OR REPLACE FUNCTION udf.num_weeks_gte_n_from_bits(b INT64, n INT64) AS (
```

The 28-day bit patterns in clients_last_seen are integers rather than BYTES.